### PR TITLE
Rosetta entropy

### PR DESCRIPTION
**Pull request by privat**

### CODE REVIEW

**Comment by R4PaSs** [comment]
:shipit:

**Comment by privat** [comment]
update: also add entropy_narcissist as suggested by lucas (just a one-liner)

**Comment by R4PaSs** [comment]
@privat : Why is the narcissist's entropy 0.0 ?

**Comment by privat** [comment]
@R4PaSs because wrong directory, so read_all returns "" that has en entropy of 0

**Comment by privat** [comment]
update: also add fibonacci_word because related to entropy and easy to write

**Comment by R4PaSs** [comment]
Makes sense ! :shipit:

File: examples/rosettacode/entropy_narcissist.nit [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# This program is public domain
+
+# Task: Entropy/Narcissist
+# SEE: <http://rosettacode.org/wiki/Entropy/Narcissist>
+module entropy_narcissist
+
+import counter
+
+# Should be run in the right directory
+print "entropy_narcissist.nit".to_path.read_all.chars.to_counter.entropy
```
Comment (Morriar) [inline examples/rosettacode/entropy_narcissist.nit]: Why not use args[0]?

Comment (privat) [inline examples/rosettacode/entropy_narcissist.nit]: args[0] will be the first argument, and program_name will be the name of the binary

Comment (Morriar) [inline examples/rosettacode/entropy_narcissist.nit]: Alrigth then

**Comment by Morriar** [comment]
+1

**Pull request merged**